packages/0xcert-conventions/src/assets/86-base-asset.ts

```diff
@@ -4,8 +4,8 @@ import { ObjectErc721, schemaErc721 } from './erc721';
  * Base asset data object interface.
  */
 export interface Object86 extends ObjectErc721 {
-  $evidence?: string;
```

Comment:
This is an interface, there are no private/protected variables here so there should be no change here.